pages/bookshelf.tsx

```diff
@@ -0,0 +1,65 @@
+import { NextPage } from "next"
+import "react-toastify/dist/ReactToastify.css"
+import Navigation from "../components/bookshelf/Navigation"
+import { Container } from "../components/styled/bookshelf"
+import { useState, useContext, useEffect, useMemo } from "react"
+import Discover from "../components/bookshelf/discover/Discover"
+import LogoutBox from "../components/bookshelf/LogoutBox"
+import { UserContext } from "../context/UserContext"
+import { useRouter } from "next/router"
+import { getBooks } from "../api/books"
+import { BooksContext } from "../context/BooksContext"
+import { ActivePageEnum } from "../api/enums"
+
+const Home: NextPage = () => {
+  const [active, setActive] = useState<ActivePageEnum | string>(
+    ActivePageEnum.to_read
+  )
+  const router = useRouter()
+
+  useEffect(() => {
+    const existingUser = JSON.parse(localStorage.getItem("user") || "null")
```

Comment:
`nit`

"null" looks weird. I know you're parsing it via json.

```suggestion
    const existingUser = localStorage.getItem("user") && JSON.parse(localStorage.getItem("user"))
```

Reply:
got rid of this through implementing the layout 😁